pages/index.tsx

```diff
@@ -1,12 +1,16 @@
+import { GetServerSideProps } from 'next';
 import { useRouter } from 'next/router';
 import { useState } from 'react';
 
 import HomeSearchBar from '../components/HomeSearchBar';
 import Layout from '../components/Layout';
+import { API_URL } from '../lib/utils';
 
-export default function HomePage() {
+export default function HomePage({ packageCount }) {
```

Comment:
It's nice to add the type of props, you can just do it inline here since it's only 1 prop
```suggestion
export default function HomePage({ packageCount }: { packageCount?: number }) {
```

Reply:
The header returns a string, should I cast it to a number or change propType?